build.gradle

```diff
@@ -77,3 +77,4 @@ artifacts {
 }
 
 apply from: file('gradle/gradle-mvn-push.gradle')
+apply from: file('gradle/quality.gradle')
```

Comment:
let's go and move everything inside here. just restore the top line:

```diff
-apply plugin: 'com.vanniktech.android.junit.jacoco'
```

and add the content of the `quality.gradle` file